### ISSUE TITLE
Main navigation color contrast is not high enough

### ISSUE THREAD

**Issue by knice** [ucsc/wcms-design-templates#35]
Without light background, the contrast between the gold color and the background does not meet accessibility standards.

![2014-06-12-144805](https://cloud.githubusercontent.com/assets/36719/3264163/5a45aad0-f27b-11e3-9b06-00d9515352a3.png)
